features/shop/cart_with_different_taxes_for_different_zones.feature

```diff
@@ -0,0 +1,50 @@
+@ui-cart
+Feature: Cart with different tax rates for different time zones
+    In order to order products with different tax rate for different zones
+    As a Customer
+    I want to be aware of taxes applied on my order
+
+    Background:
+        Given that store is operating on the France channel
+        And default currency is "EUR"
+        And store has "EU VAT" tax rate of 23% for "Clothes" in "EU" zone
+        And store has "No tax" tax rate of 0% for "Clothes" in "Rest of World" zone
+        And store has "Low VAT" tax rate of 5% for "Mugs" in "EU" zone
+        And there is user "john@example.com" identified by "password123"
+        And catalog has a product "PHP T-Shirt" priced at €100.00
```

Comment:
This step has been based on previously written payment scenario by @Arminek, but sure, I think it's more natural